app/models/limber/plate.rb

```diff
@@ -29,16 +29,13 @@ def purpose
   # We know that if there are any transfers with this plate as a source then they are into
   # tubes.
   def transfers_to_tubes?
-    transfers_to_tubes.present?
+    transfer_request_collections.present? || transfers_to_tubes.present?
   end
 
   def tubes_and_sources
```

Comment:
rename to 'tubes_and_source_wells' (?)
and maybe add some documentation on what it returns and what it is doing?